_drafts/2017-07-27-taxonomy-suite.md

```diff
@@ -45,7 +46,7 @@ We have an ever-growing suite of taxonomy packages.
 
 * `taxize` - taxonomic data from many sources
 * `taxizedb` - work with taxonomic SQL databases locally
-* `taxa` - taxonomic classes for R
+* `taxa` - taxonomic classes and manipulation functions
 * `binomen` - (getting folded into `taxa`, will be archived on CRAN soon)
 * `wikitaxa` - taxonomic data from Wikipedia/Wikidata/Wikispecies
 * `ritis` - get ITIS (Integrated Taxonomic Information Service) taxonomic data
```

Comment:
ok, was not aware of "badge styled object". I guess it's kind of jargony.

Still " (cran if on CRAN, cran if not)" is one cran supposed to be linked?